.github/workflows/pdf-docs-generation.yml

```diff
@@ -0,0 +1,24 @@
+name: PDF Docs Generation
+
+on:
+  push:
```

Comment:
For now yes, why not. since we are not pushing anything to repo as the by products from the pipeline are extraterrestrials for the repo. static site pipeline sends files directly to github pages. and products doc pdfs sit in the actions pipeline.